update jmx2logzio readme (#262)
logzio/logz-docs: _source/logzio_collections/_metrics-sources/jmx.md

## jmx2logzio + Jolokia agent setup

> If you to poll metrics from a remote source, you can use Jolokia agent + Jmx2logzio

Reviewer: ```suggestion
If you want to poll metrics from a remote source, you can use Jolokia agent + Jmx2logzio.
```

> In this configuration, Jolokia exposes the metrics as REST, and jmx2logzio reads it from them and sends it to Logz.io

Reviewer: Not sure what you mean here—can you clarify?

Author: The previous description was wrong and misleading. What's actually happens is Jolokia polls the metrics from the target app, expose them as REST API, and once every x seconds jmx2logzio call this api and receive the metrics as answer

Reviewer: ```suggestion
In this configuration, Jolokia exposes the metrics through an API, which jmx2logzio reads and sends to Logz.io.
```
